org.gnu.Octave.json

```diff
@@ -0,0 +1,186 @@
+{
+	"app-id": "org.gnu.Octave",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "master",
+	"sdk": "org.kde.Sdk",
+	"command": "octave",
+	"rename-icon": "octave",
+	"rename-desktop-file": "www.octave.org-octave.desktop",
+	"rename-appdata-file": "www.octave.org-octave.appdata.xml",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--device=dri",
+		"--filesystem=home"
+	],
+	"build-options": {
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/lib/debug",
+		"/lib/cmake",
+		"*.a",
+		"*.la"
+	],
+	"modules": [
+		{
+			"name": "gmp",
+			"config-opts": [
+				"--enable-shared",
+				"--disable-static"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://gmplib.org/download/gmp/gmp-6.1.2.tar.xz",
+					"sha256": "87b565e89a9a684fe4ebeeddb8399dce2599f9c9049854ca8c0dfbdea0e21912"
+				}
+			]
+		},
+		{
+			"name": "mpfr",
+			"config-opts": [
+				"--enable-shared",
+				"--disable-static"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://www.mpfr.org/mpfr-current/mpfr-3.1.5.tar.gz",
+					"sha256": "f4eb5070883aee3fd8b927751ea63ff95aebe24418cde852439ce74c3dd2513c"
+				}
+			]
+		},
+		{
+			"name": "mpc",
+			"config-opts": [
+				"--enable-shared",
+				"--disable-static"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://ftp.gnu.org/gnu/mpc/mpc-1.0.3.tar.gz",
+					"sha256": "617decc6ea09889fb08ede330917a00b16809b8db88c29c31bfbb49cbf88ecc3"
+				}
+			]
+		},
+		{
+			"name": "gcc-fortran",
+			"config-opts": ["--enable-languages=fortran", "--disable-multilib"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://mirrors.kernel.org/gnu/gcc/gcc-7.1.0/gcc-7.1.0.tar.gz",
+					"sha256": "3c187ef42e4cf657eec0ed641aac7f7e46261bcb1cea31b72dcf2dedddbbdea5"
+				}
+			]
+		},
+		{
+			"name": "gfortran-fix",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "shell",
+					"commands": ["echo 'Symlinking libgfortran.so.4'"]
+				}
+			],
+			"build-commands": ["ln -s /app/lib64/libgfortran.so.4 /app/lib/libgfortran.so.4", "ln -s /app/lib64/libquadmath.so.0 /app/lib/libquadmath.so.0"]
```

Comment:
Alternatively, use something like the the extension does:
                    "commands": ["sed -e '/m64=/s/lib64/lib/' -i.orig gcc/config/i386/t-linux64"]
Which i believe should make it install in /lib directly.